spec/predicates/human-review-vcs.md

```diff
@@ -0,0 +1,113 @@
+threshold of approvals before a pull request can be merged. This predicate can
+be used to express meeting such a policy. In this case, GitHub would issue a
+signed attestation identifying all the reviewers for a pull request. The
+subject of the attestation is the underlying commit at the tip of the feature
+branch. This commit, therefore, is one of the parents of the merge commit if it
+is created. If an option that requires rebasing (squash and merge / rebase and
+merge) is selected, the newly created tip commit is used as the attestation
+subject.
+
+## Model
+
+Typically, software development processes incorporate some type of code review.
+Other members of the development team are expected to study the changes
+proposed and approve them, confirming they meet various expectations pertaining
+to code quality, functionality, and so on. Therefore, the general expectation
+is that this attestation is generated as a confirmation, an approval of some
+specific change, and is performed by developers in the supply chain who have
+code review responsibilities.
+
+As this attestation is generated by a version control service or code review
+service, the signatures are from the service rather than the specific users
+issuing the review.
+
+## Schema
+
+```json
+{
+    "_type": "https://in-toto.io/Statement/v1",
+    "subject": [
+        {
+            "name": "<REPO>",
+            "digest": { "gitCommit": "<HASH>" },
+            "uri": "<LINK TO REPO>"
+        },
+    ],
+    "predicateType": "http://in-toto.io/attestation/human-review/vcs/v0.1",
+    "predicate": {
+        "reviewer": ["<REVIEWER>", ...],
+        "reviewLink": "<LINK TO REVIEW>",
+        "timestamp": "<TIMESTAMP>"
+    }
+}
+```
+
+### Parsing Rules
+
+This predicate follows the
+[in-toto Attestation Framework's parsing rules](../v1/README.md#parsing-rules).
+
+### Fields
+
+`reviewer` _array of reviewer objects, _required_
+
+Indicates uniquely the identity of one or more reviewers. MUST NOT be used over
+the attestation's signature to validate the source of the attestation. This
+field supports scenarios where the attestation is issued and signed by the VCS
+system. Each entry must include immutable information that is always mapped to
+the reviewer. The reviewer object is not explicitly defined as it specific to
+the VCS system.
+
+`reviewLink` _URI_, _optional_
+
+Contains a link to the full review. Useful to point to information that cannot
+be captured in the attestation.
+
+`timestamp` _Timestamp_, _optional_
+
+Indicates time of review creation.
+
+## Example
+
+```json
+{
+    "_type": "https://in-toto.io/Statement/v1",
+    "subject": [{
+        "name": "in-toto-python",
+        "digest": {
+            "gitCommit": "b0bd8ecab5607e174fa403e002c74a666e7edd51"
+        },
+        "uri": "https://github.com/in-toto/in-toto"
+    }],
+    "predicateType": "http://in-toto.io/attestation/human-review/vcs/v0.1",
+    "predicate": {
+        "reviewer": [{
```

Comment:
This does not follow the `REVIEWER` types?

Reply:
This was previously set to be opaque per system but now uses RD.